sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -362,6 +365,8 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
    */
   companion object {
     internal const val DEFAULT_ANTIALIASING_SAMPLE_COUNT = 1
+    internal const val MAX_POSSIBLE_FPS = 1000L
```

Comment:
why 1000L?

Reply:
just some big value (who knows what screens will appear on devices in the nearest future) - but not big enough (as let's say MAX_INT that breaks the code). Setting 1000 as max fps will actually simply mean that device will render with max possible speed (most likely 60, 90 or 120 fps).

Reply:
Why do we even need a max possible fps if the maximum is capped with the device's supported refresh rate? The argument is long type, I guess that's enough already?

Reply:
cause user could potentially pass MAX_INT and break all the things, that's why I did add some limits